client/my-sites/domains/domain-management/dns/a-record.jsx

```diff
@@ -67,6 +68,24 @@ class ARecord extends Component {
 					/>
 					{ ! isDataValid && <FormInputValidation text={ translate( 'Invalid IP' ) } isError /> }
 				</FormFieldset>
+
+				<FormFieldset>
+					<FormLabel>TTL</FormLabel>
```

Comment:
```suggestion
					<FormLabel>TTL (time to live)</FormLabel>
```
I wonder if adding this would be helpful, but I guess users who need to change this would already know what it is 😄

Reply:
Males sense :)